exercises/bracket-push/tests/bracket-push.rs

```diff
@@ -1,88 +1,88 @@
 extern crate bracket_push;
 
-use bracket_push::*;
+use bracket_push::brackets_are_balanced;
```

Comment:
note: If the crate name were `brackets`, I would have wanted the name to be `brackets::are_balanced` to avoid repetition. However, looks like we are using `bracket_push`. So I think `brackets_are_balanced` has to be the name.

 It can be worth a discussion whether to change `bracket_push` (why is it even called bracket-push??? https://github.com/exercism/problem-specifications/issues/693 ) but this PR does not need to depend on it.